### ISSUE TITLE
Drawing on the plate moves to other plates

### ISSUE THREAD

**Issue by n251ea** [apps4av/avare#356]
When drawing on a plate, the drawing will display on any plate displayed, for example if I circle the departure procedures that are relevant to me and then switch to the airport diagram, the drawing will be on the airport diagram (or any other plate) too.

![Screenshot_20191103-100907](https://user-images.githubusercontent.com/48159367/68087283-487cea80-fe22-11e9-9542-846eec95e187.jpg)
![Screenshot_20191103-100850](https://user-images.githubusercontent.com/48159367/68087284-487cea80-fe22-11e9-9991-f6fb7ed7627f.jpg)

**Comment by reddn** [comment]
Does the feature state the drawing will stay only on the selected plate?

**Comment by n251ea** [comment]
I don't think it states anything, but carrying the drawing to other plates is pointless

**Comment by reddn** [comment]
I think that depends on the use case.  If you are looking at a STAR and ATC
gives you a route and you use that feature to write it on there, if you
change plates or anything, you lose it.
